nntrainer/tensor/cl_operations/blas_kernels_templates.h

```diff
@@ -258,38 +258,56 @@ sgemm_cl_internal(ClContext::SharedPtrClKernel kernel, bool TransA, bool TransB,
 template <typename T>
 inline static void
 addition_cl_internal(ClContext::SharedPtrClKernel kernel, const T *input,
-                     T *res, unsigned int size_input, unsigned int size_res) {
+                     T *res, unsigned int size_input, unsigned int size_res,
+                     const bool use_svm) {
   bool result = false;
 
   auto *blas_cc =
     static_cast<ClContext *>(Engine::Global().getRegisteredContext("gpu"));
-  auto &clbuffInstance = ClBufferManager::Global();
 
   size_t dim1_size = sizeof(T) * size_input;
   size_t dim2_size = sizeof(T) * size_res;
 
-  result = clbuffInstance.getInBufferA()->WriteDataRegion(
-    blas_cc->command_queue_inst_, dim1_size, input);
-  if (!result) {
-    return;
-  }
+  if (use_svm) {
+    blas_cc->command_queue_inst_.enqueueSVMMap(const_cast<float *>(input),
+                                               dim1_size, false);
+    blas_cc->command_queue_inst_.enqueueSVMMap(res, dim2_size, false);
```

Comment:
This is incorrect. SVMMap is used to enable the **host side** to use the allocated SVM.
```suggestion
    blas_cc->command_queue_inst_.enqueueSVMUnmap(input);
    blas_cc->command_queue_inst_.enqueueSVMUnmap(res);
```